gnmi_cli_py/py_gnmicli.py

```diff
@@ -430,8 +441,15 @@ def subscribe_start(stub, options, req_iterator):
               print('gNMI Error '+str(response.error.code)+\
                 ' received\n'+str(response.error.message) + str(response.error))
           elif response.HasField('update'):
-              print(response)
-              update_count = update_count+1
+              if filter_event != "":
+                  if filter_event in str(response):
+                      if event_op_file != "":
+                          handle_event_response(response, event_op_file)
```

Comment:
>handle_event_response [](http://example.com/codeflow?start=26&length=21)

Seems like the option `--event_op_file` is not necessary, you always print to stdout, then you can redirect it to a file.